statsd/telemetry.go

```
@@ -0,0 +1,307 @@
+	AggregationNbContextTiming uint64
+}
+
+type telemetryClient struct {
+	sync.RWMutex // used mostly to change the transport tag.
+
+	c                 *Client
+	aggEnabled        bool // is aggregation enabled and should we sent aggregation telemetry.
+	transport         string
+	tags              []string
+	tagsByType        map[metricType][]string
+	transportTagKnown bool
```

golangci-lint (1.62.2)

field `transportTagKnown` is unused

(unused)